be/src/exec/vectorized/schema_scanner/schema_task_runs_scanner.cpp

```diff
@@ -139,9 +140,29 @@ Status SchemaTaskRunsScanner::fill_chunk(ChunkPtr* chunk) {
             break;
         }
         case 8: {
-            // ERROR_CODE
+            // EXPIRE_TIME
             {
                 ColumnPtr column = (*chunk)->get_column_by_slot_id(8);
+                auto* nullable_column = down_cast<NullableColumn*>(column.get());
+                if (task_run_info.__isset.expire_time) {
+                    int64_t expire_time = task_run_info.expire_time;
+                    if (expire_time <= 0) {
+                        nullable_column->append_nulls(1);
+                    } else {
+                        DateTimeValue t;
+                        t.from_unixtime(expire_time, TimezoneUtils::default_time_zone);
```

Comment:
nit: how about adding a function for this? since finish_time create_time will also use this

Reply:
You're right, but I plan to do it later.